### ISSUE TITLE
RPPA data issue for lihc_tcga

### ISSUE THREAD

**Issue by jjgao** [cBioPortal/datahub#421]
As [reported by a user](https://groups.google.com/forum/#!msg/cbioportal/u-4VpoPN-lg/N71oJr0zCgAJ), the samples in the RPPA data does not seems like liver tumors. Is there a mixed up?

**Comment by ritikakundra** [comment]
@jjgao Checked back the raw provisional data files in our staging directory from 2016. Looks like RPPA files always had the 63 samples for the liver but on checking in GDC they don't look like liver cases. Am going to recheck mercurial history again to confirm.

**Comment by jjgao** [comment]
@ritikakundra Thanks. The problem is that the samples are not liver samples.

**Comment by ritikakundra** [comment]
@jjgao Yes I got that, but I wanted to check if this was a data issue from our end or did these come in from the raw source. And from what I see from the raw 2105 analysis file (gdac.broadinstitute.org_LIHC.RPPA_AnnotateWithGene.Level_3.2015082100.1.0.tar.gz), all the RPPA samples are not liver. We might have to check all provisionals for such discrepancies.
